<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResetMedicalDataWindow</class>
 <widget class="QMainWindow" name="ResetMedicalDataWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="applyPushButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>570</y>
      <width>761</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Оновити довідку</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>761</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Оновлення медичної довідки&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>761</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="doctor">
       <property name="text">
        <string>ПІБ лікаря</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="enddate">
       <property name="text">
        <string>Дата кінця дії</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="startdate">
       <property name="text">
        <string>Дата початку дії</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDateEdit" name="startDateEdit">
       <property name="displayFormat">
        <string>yyyy.MM.dd</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="doctorLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="decision">
       <property name="text">
        <string>Висновок лікаря</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="institution">
       <property name="text">
        <string>Заклад, що видав</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="decisionLineEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="institutionLineEdit"/>
     </item>
     <item row="3" column="2">
      <widget class="QDateEdit" name="endDateEdit">
       <property name="displayFormat">
        <string>yyyy.MM.dd</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>934</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
